runtime/sema/member_accesses.go

```diff
@@ -67,11 +63,7 @@ func (m *MemberAccesses) Find(pos Position) *MemberAccess {
 func (m *MemberAccesses) All() []MemberAccess {
 	values := m.tree.Values()
 	accesses := make([]MemberAccess, len(values))
-	for i, value := range values {
-		access, ok := value.(MemberAccess)
-		if !ok {
-			continue
-		}
+	for i, access := range values {
```

Comment:
There's probably no need anymore to create a copy, the result of `Values` should already be `[]MemberAccess` now, right?